ls1intum/Artemis
# [Exam] Client: Main Component

### Checklist

### Motivation and Context


Set up the main component and structure to start properly developing the exam mode client side.
https://github.com/ls1intum/Artemis/issues/1568

### Description

I initialized the directory and component structure and routes for the exam mode
Routes:
- courses/{courseId}/exam: this routing scheme is only temporal and will be extended by the student's participation. The student's exam view will be loaded here.
- courses/{courseId}/exams: students' overview over the exams for the course
- course-management/{courseId}/exam: This is the instructor's overview. The creation and management of the exams will take place here.

### Steps for Testing


1. Log in to Artemis
2. Check that courses/{courseId}/exam is working
2. Check that courses/{courseId}/exams is working
2. Check that course-management/{courseId}/exams is working

### Screenshots
